### PR TITLE
[5.7] Remove unnecessary "/home" path directory when running "make:auth" command.

### PR DESCRIPTION
**Pull request by GabMic**
there is no need for this path. a regular `/` path like any other site worldwide is the better option.

### CODE REVIEW

**Comment by driesvints** [comment]
Like I said in https://github.com/laravel/laravel/pull/4819, I think you're misunderstanding what `/home` stands for. Please have a look at my remarks there.

File: src/Illuminate/Auth/Console/stubs/make/routes.stub [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
+Route::get('/', 'HomeController@index')->name('home');
```
Comment (driesvints) [inline src/Illuminate/Auth/Console/stubs/make/routes.stub]: This will conflict with the base laravel skeleton which already has a `/` route defined.

Comment (GabMic) [reply src/Illuminate/Auth/Console/stubs/make/routes.stub]: Check what i have said in here  laravel/laravel#4819 #26134

**Comment by jmarcher** [comment]
Just leave it as it is, this can cause more confusion than help newcomers.

**Comment by GabMic** [comment]
Newcomers will just follow. I bet you don't even use the home directory.  And of you do, i literally can't understand why.

**Comment by jmarcher** [comment]
Do not place your bets wrong :) I do not think you understand that there are 2 routes pointing to "/" as a newcomer this will cause confusion.

**Comment by 36864** [comment]
This is a breaking change and would at a minimum need to target master instead of 5.7

As for using `/home`, I do use it in a couple of projects. No particular reason other than wanting a clear separation between the landing page and a user's home page. The root URL ( `/` ) shows unfiltered content, while `/home` shows personalized content.

**Pull request closed without merging**